pkg/operation/botanist/health_check.go

```diff
@@ -616,10 +617,14 @@ func (b *HealthChecker) CheckLoggingControlPlane(
 }
 
 // CheckExtensionCondition checks whether the conditions provided by extensions are healthy.
-func (b *HealthChecker) CheckExtensionCondition(condition gardencorev1beta1.Condition, extensionsCondition []extensionCondition) *gardencorev1beta1.Condition {
-	for _, cond := range extensionsCondition {
-		if cond.condition.Status == gardencorev1beta1.ConditionFalse || cond.condition.Status == gardencorev1beta1.ConditionUnknown {
-			c := b.FailedCondition(condition, fmt.Sprintf("%sUnhealthyReport", cond.extensionType), fmt.Sprintf("%q CRD (%s/%s) reports failing healthcheck: %s", cond.extensionType, cond.extensionNamespace, cond.extensionName, cond.condition.Message))
+func (b *HealthChecker) CheckExtensionCondition(condition gardencorev1beta1.Condition, extensionsConditions []ExtensionCondition) *gardencorev1beta1.Condition {
+	for _, cond := range extensionsConditions {
+		if Now().UTC().Sub(cond.Condition.LastUpdateTime.UTC()).Minutes() > b.healthCheckOutdatedThreshold.Duration.Minutes() {
```

Comment:
Hm, why is this conversion to minutes needed?

Reply:
It does not really matter, it is just to be able to compare to floats. Changed it to seconds. 
In the condition message I think having it as rounded to minutes makes sense to be more readable.

Reply:
Hm, are we doing this anywhere else? I can't recall I've seen it before. It should not be needed, or?

Reply:
You are right, we can just compare the durations with converting them to minutes / second. Thanks.